docs/management_api_v2.yml

```diff
@@ -0,0 +1,507 @@
+swagger: '2.0'
+info:
+  version: '2'
+  title: Device authentication
+  description: |
+      An API for device authentication handling.
+
+basePath: '/api/management/v2/devauth/'
+host: 'mender-device-auth:8080'
+schemes:
+  - http
+
+paths:
+  /devices:
+    get:
+      summary: Get a list of tenant's devices.
+      description: |
+        Provides a list of tenant's devices, with optional device status filter.
```

Comment:
Is this list sorted? It might be useful to sort by created_ts or updated_ts

Reply:
currently, in the deviceauth and deviceadm services, we are sorting devices by id
to me, it sounds reasonable to sort them by created_ts
@mchalski @maciejmrowiec what do you think?

Reply:
👍